# gate_pos_vec: [
# -5.44727,-0.662101, 1.26864, # ring_2
# -0.831071,0.546521, 0.820996, # ring_6
# 4.41507,-0.711206, 1.37091, # ring_4
# 5.0, 0.8, 1.0,  # fake
# 3.56527,2.64906, 0.874866, # ring_3
# -0.892368,0.566543, 1.77146, # ring_5
# -5.88727,2.38504,0.851399, # ring_1
# -6.5, 1.8, 1.0,  # fake
# -5.44727,-0.662101, 1.26864, # ring_2
# -0.831071,0.546521, 0.820996, # ring_6
# 4.41507,-0.711206, 1.37091, # ring_4
# 5.0, 0.8, 1.0,  # fake
# 3.56527,2.64906, 0.874866, # ring_3
# -0.892368,0.566543, 1.77146, # ring_5
# -5.88727,2.38504,0.851399, # ring_1
# ]

# gate_oritation: [
# -0.04, # ring_2
# -0.07,  # ring_6
# -0.04, # ring_4
# 1.57,  # fake
# 3.08743, # ring_3
# 3.0687, # ring_5
# 3.09474, # ring_1
# -1.57, #fake
# -0.04, # ring_2
# -0.07,  # ring_6
# -0.04, # ring_4
# 1.57,  # fake
# 3.08743, # ring_3
# 3.0687, # ring_5
# 3.09474, # ring_1

# ]

# target_pos: [
#   -7.0, 2.8, 0.9
# ]

# gate_radius: 0.4
# inner_gate_radius:    0.4
# outter_gate_radius:   0.55

# special_gate_index: [
#  3, 7, 11,
# ]

# initial_pos: [
# -7.5, -1.0, 1.0]

#------------------------------------------------------------------------------------------
# gate_pos_vec: [
# -3.0, -1.5, 1.5,
# 0.5, -0.5, 0.6,
# 4.5, -1.5, 1.5,
# 5.5, -0.5, 1.5, # fake 3
# 3.5, 0.5, 1.5,
# 0.5, -0.5, 1.7,
# -4.5, 0.5, 1.5,
# # -5.5, -0.5, 1.5,  # fake 7 # first loop over
# # -3.0, -1.5, 1.5,
# # 0.5, -0.5, 0.6,
# # 4.5, -1.5, 1.5,
# # 5.0, -0.5, 1.5, # fake 11
# # 3.5, 0.5, 1.5,
# # 0.5, -0.5, 1.7,
# # -4.5, 0.5, 1.5, # second loop over
# ]

# gate_oritation: [
#   0, 0, 0, 1.57, 3.14, 3.14, 3.14, # first loop over
#   # -1.57, 0, 0, 0, 1.57, 3.14, 3.14, 3.14, # second loop over
# ]

# target_pos: [
#   # -5, -1.5, 1.5
#   -5.5, 0.5, 1.5
# ]

# gate_radius: 0.4
# inner_gate_radius:    0.4
# outter_gate_radius:   0.55

# special_gate_index: [
#   3, 7, # first loop over
#   # 11, # second loop over
# ]

# initial_pos: [
# -5, -1.5, 1.5]

#------------------------------------------------------------------------------------------
# # zyh designed 2
# gate_pos_vec: [
# 15.0, -1.0, 3.0, #gaste1
# 25.0, -8.0, 2.0, # gate2
# 35.0, -17.0, 1.3,
# 30, -29.0, 1.2,
# 27.5, -25.0, 1.2,
# 25, -21.0, 1.2,
# 22.5, -25.0, 1.2,
# 20, -29.0, 1.2,
# 17.5, -25.0, 1.2,
# 15, -21.0, 1.2,
# 10.0, -25.0, 1.0,
# -5.0, -15.0, 1.34,
# -10.0, 0.0, 5.0,
# ]

# gate_oritation: [
#   0, 0, -1.57, 3.14, 1.57, 3.14, -1.57, 3.14, 1.57, 3.14, 3.14, 1.57, 1.57,
# ]

# target_pos: [
#   -10.0, 1.0, 4.0,
# ]

# special_gate_index: [
#   3, 4, 5, 6, 7, 8 , 9 # first loop over
#   # 11, # second loop over
# ]

# gate_radius: 1.0
# inner_gate_radius:    1.0
# outter_gate_radius:   1.2

# initial_pos: [
#   0, 0, 1.0,
# ]

#------------------------------------------------------------------------------------------
# zyh designed 2
gate_pos_vec: [
8.0, -1.0, 1.2, #gaste0
17.0, -1.0, 3.0, #gaste1
# 23.0, 1.0, 2.0, # gate2
28.0, 0.0, 2.0, # fake
35.0, -2.0, 2.0, # gate3
38.0, -7.0, 2.0, # fake
35.0, -13.0, 2.0, # gate411
30, -9.0, 2.0, # fake
26.5, -13.0, 2.0, # fake
23, -17.0, 2.0, # fake
19.5, -13.0, 2.0, # fake
15.0, -9.0, 2.0, # fake
3.5, -12.0, 2.0, # gate
3, -16.5, 4.0, # fake
4, -24.0, 6.0, # fake
13, -25.0, 6.0, # gate5
20, -22.0, 7.0, # gate6
# 28, -26.0, 6.0, # gate7
35, -25.0, 6.0, # gate8
]

gate_oritation: [
  0, 0, 0, 0, -1.57, 3.14, 3.14, -1.57, 3.14, 1.57, 3.14, 4.0, -1.57, 0, 0, 0, 0, 
]

target_pos: [
  37 , -25, 5.0,
]

special_gate_index: [
  4, 6, 7, 8, 9, 10, 12, 13, # first loop over
  # 11, # second loop over
]

gate_radius: 1.0
inner_gate_radius:    1.0
outter_gate_radius:   1.2

initial_pos: [
  0, 0, 1.0,
]
